packages/config/lib/index.js

```diff
@@ -101,4 +101,20 @@ module.exports = {
       }
     })
   },
+
+  validateNoReadOnlyConfig: (config, onErr) => {
+    const writeableOptions = options.filter((option) => option.isWriteable).map((option) => option.name)
```

Comment:
If options is static, we can declare writeableOptions outside of the function's scope and save a few iterations per call. It'd fit in pretty well with the `breakingKeys`/`defaultValues` vars declared above.